winter_messaging_transactional/rabbitmq/message_listener.py

```diff
@@ -0,0 +1,91 @@
+import json
+import logging
+from uuid import UUID
+
+from injector import inject
+from pika import BasicProperties
+from pika.adapters.blocking_connection import BlockingChannel
+from pika.spec import Basic
+from sqlalchemy.engine import Engine
+from winter.core.json import json_decode
+from winter.messaging import Event
+from winter.messaging import EventHandlerRegistry
+from winter.messaging.event_dispacher import EventDispatcher
+from winter_messaging_transactional.consumer.event_processing_logger import EventProcessingLogger
+from winter_messaging_transactional.consumer.inbox.inbox_message import InboxMessage
+from winter_messaging_transactional.consumer.inbox.inbox_message_dao import InboxMessageDAO
+from winter_messaging_transactional.consumer.middleware_registry import MiddlewareRegistry
+from winter_messaging_transactional.consumer.timeout_handler import TimeoutHandler
+
+logger = logging.getLogger(__name__)
+
+EVENT_HANDLING_TIMEOUT = 15
+RETRIES_ON_TIMEOUT = 1
+
+
+class MessageListener:
+    MAX_RETRIES = 3
```

Comment:
nit: Some variables are outside of class. I think it's better to make all of them class parameters.